docs/install_guides/installing-model-dev-tools.md

```diff
@@ -91,65 +91,67 @@ Note that the commands are different for a Python installation and a C++ install
 
 .. tab:: Python
 
-  To install and configure the components of the development package for working with specific frameworks, use the following command:
+   To install and configure the components of the development package for working with specific frameworks, use the following command:
 
    .. code-block:: sh
-   
+
       pip install openvino-dev[extras]
-      
-  where the `extras` parameter specifies one or more deep learning frameworks via these values: `caffe`, `kaldi`, `mxnet`, `onnx`, `pytorch`, `tensorflow`, `tensorflow2`. Make sure that you install the corresponding frameworks for your models.
-  
-  For example, to install and configure the components for working with TensorFlow 2.x and ONNX, use the following command:
-  
+
+   where the `extras` parameter specifies one or more deep learning frameworks via these values: `caffe`, `kaldi`, `mxnet`, `onnx`, `pytorch`, `tensorflow`, `tensorflow2`. Make sure that you install the corresponding frameworks for your models.
```

Comment:
Recently, we came up with a different approach to how to use extras for openvino-dev package. The main idea is to provide a smooth transition from DL framework to openvino. Here is how it should work:
- The user trains/downloads a model in the framework representation (99+%) it is about the Python environment.
- Using a framework model means that a specific version of the framework is installed already.
- The user decides to use OpenVINO framework. To do that it installs openvino-dev package into the same python environment where the framework is installed. No extras are used in this case.
- In case there are any issues with framework-openvino compatibility the user can try to install openvino-dev into a separate environment and use extra to get the specific validated version of the framework.

Hope this approach is clear. If so, can you please to revise this section accordingly?

Reply:
Thanks for the update @AlexKoff88 ! 
Can we revise this section like this:

- If you have installed your deep learning framework in the same environment as OpenVINO Runtime, run `pip install openvino-dev` to install OpenVINO Development Tools.
- In case that you encounter any compatibility issues between OpenVINO and your deep learning framework, you may install OpenVINO Development Tools into a separate environment and use the following command to get specific validated versions of your framework:
    .. code-block:: sh
 
       pip install openvino-dev[extras]

  where [extras]...

Reply:
And this only affect the Python installation, right? Do we also need to change the recommendations for C++ users?

Reply:
I would rephrase the first part as follows:
* You can install OpenVINO Developer Tools into the existing environment with the deep learning framework of your choice. Run the following command to do that:
```python
pip install openvino-dev
```

Just to be on the same page. The transition from the DL framework starts not from OpenVINO Runtime. You should convert the model first using Developer Tools. Please consider this point.
BTW, dev tools usually stand for Developer Tools, not Development. Hope, we can have the same notation.

Reply:
> BTW, dev tools usually stand for Developer Tools, not Development. Hope, we can have the same notation.

I have to argue that it is Development Tools in the Intel name database. :)
<img width="318" alt="image" src="https://user-images.githubusercontent.com/95474734/206666574-94fe3c2a-f647-4009-b305-c4437628ae64.png">

Reply:
Revision applied with https://github.com/openvinotoolkit/openvino/pull/14492/commits/45f952fd3ddd82d0eaa305e1a09e5511169b7d44